src/build/emitter.ts

```diff
@@ -387,17 +388,51 @@ export function emitWebIdl(
             : obj.type;
         types.push(...(obj.additionalTypes ?? []).map((t) => ({ type: t })));
 
+        // propagate `any`
         const converted = types.map(convertDomTypeToTsTypeWorker);
-        const isAny = converted.some((t) => t === "any");
-        return isAny ? "any" : converted.join(" | ");
+        if (converted.includes("any")) return "any";
```

Comment:
Nice change but would be nice to return in a block for style consistency. (Maybe there should be an eslint config for this)